### ISSUE TITLE
Error "ModuleNotFoundError: No module named 'oggm'", PyCharm, macOS (Big sur) 

### ISSUE THREAD

**Issue by Sayantan-1996** [OGGM/tutorials#45]
Hi all,
I am new to OGGM. I have to start my project, so I was setting up OGGM on my mac. I was following this tutorial from OGGM website [Getting Started](https://docs.oggm.org/en/v1.5.2/getting-started.html). I wrote the following lines of code in PyCharm:

```
from oggm import cfg
cfg.initialize()
```
Executing this throws an error:

`ModuleNotFoundError: No module named 'oggm'`

I installed OGGM as per the instruction at OGGM website [Install docs ](https://docs.oggm.org/en/v1.1/installing-oggm.html).

How to proceed from here? In worst case how to remove everything OGGM related from my machine so that I can start fresh?

Thanks in advance

**Comment by fmaussion** [comment]
Dear @Sayantan-1996 , unfortunately you are not providing enough information for us to help you out. Are the other packages required by oggm (xarray, rasterio, pandas...) installed properly? Can you do `import rasterio` for example?

**Comment by Sayantan-1996** [comment]
Dear @fmaussion , sorry I did not write my query properly earlier. I followed the instructions and downloaded the oggm dependecies using

`conda install -c oggm -c conda-forge oggm-deps`

Before this step I set up the oggm environment by 

```
conda create --name oggm_env python=3.X
source activate oggm_env
```
I must confess running

`pytest --pyargs oggm` did gave me an error

Also, I was writing the  codes:

```
from oggm import cfg
cfg.initialize()
```

 in an interpreter in the base environment. I beleive I should have started in oggm environment created as mentioned above. If that is the case how may I open the OGGM environment? (If I want to use PyCharm)
